src/Style/Sass/pages/_auth.scss

```diff
@@ -0,0 +1,59 @@
+.common-form {
+  margin: 0 auto;
+  border-radius: 0.3em;
+  box-shadow: 0 4px 4px $form-box-shadow-color;
+  color: $text-color-default;
+  height: fit-content;
+  margin: 0 auto;
+  margin-top: 50px;
+  padding: 15px;
+}
+
+.common-form-button {
+  background: $form-button-color;
+  border-radius: 40px;
+  color: $iconcolor;
+  margin: 0 auto;
+  font-size: 25px;
+  font-family: "lato"
+}
+
+.common-form-input {
+  height: 50px;
+  font-size: 18px;
+  background: #ebebeb;
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+  margin-bottom: 35px;
+}
+
+.m-3{
+  font-size: 18px;
+}
+
+.common-form-h2 {
+  font-family: Lato;
+  font-size: 30px;
+  font-style: normal;
+  font-weight: bold;
+  letter-spacing: 2px;
+  margin-bottom: 25px;
+  text-align: center;
+}
+
+@media (max-width: 360px) {
+  .common-form {
+    box-shadow: none;
+  }
+
+  .float-left {
+    text-align: center;
+  }
+
+  .float-right {
+    text-align: center;
+  }
```

Comment:
all these are bootstrap classes, can you use personalized classes instead?

Reply:
thanks I have personalized them